content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+# Sharing Experiments
+
+There are two types of remotes that can store experiments. Git remotes are
+distributed copies of the Git repository, for example on GitHub or GitLab.
+
+[DVC remotes](/doc/command-reference/remote) on the other hand are
+storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
+configure with `dvc remote`. DVC uses them to store and fetch large files that
+don't normally fit inside Git repos.
+
+DVC needs both kinds of remotes for backing up and sharing experiments.
+
+Experiment files that are normally tracked in Git (like code versions) are
+shared using Git remotes, and files or directories tracked with DVC (like
+datasets) are shared using DVC remotes.
+
+> See [Git remotes guide] and `dvc remote add` for information on setting them
+> up.
+
+[git remotes guide]:
+  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+
+Normally, there should already be a Git remote called `origin` when you clone a
+repo. Use `git remote -v` to list your Git remotes:
+
+```dvc
+$ git remote -v
+origin  https://github.com/iterative/get-started-experiments (fetch)
+origin  https://github.com/iterative/get-started-experiments (push)
+```
+
+Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+
+```dvc
+$ dvc remote list
+storage https://remote.dvc.org/get-started-experiments
+```
+
+_Q: What about neither, `cache: false` objects, and objects tracked both by DVC
+and Git?_
+
+## Uploading experiments to remotes
+
+You can upload an experiment and its files to both remotes using `dvc exp push`
+(requires the Git remote name and experiment name as arguments).
+
+```dvc
+$ dvc exp push origin exp-abc123
+```
+
+> Use `dvc exp show` to find experiment names.
+
+This pushes the necessary DVC-tracked files from the cache to the default DVC
+remote (similar to `dvc push`). You can prevent this behavior by using the
+`--no-cache` option to the command above.
+
+If there's no default DVC remote, it will ask you to define one with
+`dvc remote default`. If you don't want a default remote, or if you want to use
+a different remote, you can specify one with the `--remote` (`-r`) option.
+
+DVC can use multiple threads to upload files (4 per CPU core by default). You
+can set the number with `--jobs` (`-j`). Please note that increases in
+performance also depend on the connection bandwidth and remote configurations.
```

Comment:
Do we want to go over all the options? Maybe not need if these aren't the major use cases. It's a bit repetitive with the command reference atm.

Reply:
On the other hard we could try to remove the duplicity from the ref. and leave it here instead. Maybe it belongs more to the UG in the first place?

Reply:
I think we need these here. This is a user's guide and the user may want to learn all the available options. Ref should also include them. UG takes user's needs into consideration, Ref lists them without any sorting.

Reply:
OK. Should we make an issue to remove some of these details from the cmd ref then? I think it's OK to have some repetition but there should be a single main source of truth for the details i.e. the guide, and the ref can be pretty generic and link here for more info. That way we avoid contradicting info in the future if we forget to update one or the other.

Reply:
> Should we make an issue to remove some of these details from the cmd ref then?

Looks like it's a check box in #2548 already.